src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -89,4 +91,34 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String name} into a {@code Name}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code name} is invalid.
+     */
+    public static ModuleName parseModuleName(String name) throws ParseException {
+        requireNonNull(name);
+        String trimmedName = name.trim();
+        if (!Name.isValidName(trimmedName)) {
+            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
```

Comment:
I think we should access ModuleName class instead of Name class?

Reply:
Thank you for the comment. Agree with you, and have submitted new changes.